fetch.js

```diff
@@ -0,0 +1,70 @@
+var Bluebird  = require('bluebird');
+var _defaults = require('lodash/object/defaults');
+
+var EmberCliDeployError = require('./errors/ember-cli-deploy-error');
+
+var redis = require('then-redis');
+var redisClient;
+var defaultConnectionInfo = {
+  host: "127.0.0.1",
+  port: 6379
+};
```

Comment:
I thought more about this. I think I'm going to leave this as-is and if people mention this as confusing, I'll readdress.